framework/queue/Queue.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\queue;
+
+use yii\base\Component;
+
+/**
+ * Queue is the base class for queue classes supporting different queue back-ends.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+abstract class Queue extends Component
```

Comment:
this looks more like an interface, what are the arguments for using abstract class?